.github/workflows/CreateRelease.yml

```diff
@@ -31,6 +38,18 @@ jobs:
       uses: hyperlight-dev/ci-setup-workflow@v1.5.0
       with:
           rust-toolchain: "1.85.0"
+    - name: Verify vendor.tar
+      run: |
+        mv ./src/hyperlight_wasm/vendor.tar ./vendor.tar
+        just make-vendor-tar
+        if ! git diff --no-index ./vendor.tar ./src/hyperlight_wasm/vendor.tar; then
+          echo "vendor.tar is not up to date, please run 'just make-vendor-tar' and commit the changes"
+          exit 1
+        fi
```

Comment:
This looks to be automated for the initial release, but it will require

```
     sed '/vendor.tar/d' ./src/hyperlight_wasm/.gitignore
        git add ./src/hyperlight_wasm/vendor.tar
        git add ./src/hyperlight_wasm/.gitignore
        git commit -m "Vendor dependencies for release ${GITHUB_REF_NAME}"
```

When doing any patches on the to release branch?  Should we update the release documentation to call this out?

Reply:
IIUC, even when doing a patch release, we would still use this workflow, so we should still hit the error.
But yeah, we should add a how-to-make-releases.md in this repo similar to hyperlight core.

Reply:
Opened #88 to track this concern.

Reply:
Opened #89 to track the documentation concerns.